portfolio/src/main/webapp/script.js

```diff
@@ -38,7 +38,7 @@ function bubbleFall() {
     const id = setInterval(lowerBubbleOnePixel, 10);
     // For each movement, move the element one pixel down. At the bottom, stop moving.
     function lowerBubbleOnePixel() {
-        if (height == 1000){
+        if (height == 1000) {
             clearInterval(id);
         }
         else{
```

Comment:
thanks! Here too.

Reply:
I don't know why, but on my version that change is already completed. I will try pushing again.

Reply:
It actually won't let me commit again because it says there have been no changes.

Reply:
SG! Yeah maybe some unpushed commit or uncommited chnage?

Reply:
It will not let me commit or push again :(